### PR TITLE
*: sync with preview5

### LINKED ISSUE #24: Update to preview5
We need to check and update it to preview5 now.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #24 .

1. Replace remaining nep5 occurences with nep17.
2. Replace `onPayment` with `onNEP17Payment`.
3. Replace GAS values with integers.
4. Make contract and tx hashes more consistent across the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -440,51 +434,53 @@ the corresponding transaction information, based on the specified hash value.
...
 
 Where
 - `"jsonrpc": "2.0"` is protocol version
 - `"id": 1` is id of current request
 - `"method": "getrawtransaction"` is requested method
-- `"params": ["5ec9afdcd30b6e38280df690fb0d80490fdcd94af0c2c0bcd3f47b13114f7175", 1]` is an array of parameters, where
-   - `5ec9afdcd30b6e38280df690fb0d80490fdcd94af0c2c0bcd3f47b13114f7175` is deployment transaction hash
+  b8d9b23dbf940cfc54a631c0b177b1caf9f38bfb
+- `"params": ["3cdafad701018c6dfbd4f0703bbe4ee2a9828a427cbae1569632590
+  b8d9b23dbf940cfc54a631c0b177b1caf9f38bfb7ad402c09", 1]` is an array of parameters, where
```
Comment (roman-khimov) [inline README.md]: Not sure this string wrap displays nicely.

File: README.ru.md [hunk 2]
```diff
@@ -488,33 +482,33 @@ curl -d '{ "jsonrpc": "2.0", "id": 1, "method": "getrawtransaction", "params": [
 
 Запросите информацию о контракте для нашей вызывающей транзакции, полученной на шаге 4:
```
Comment (roman-khimov) [inline README.ru.md:482]: Deployment transaction is not updated above.

File: README.md (hunk 1 above)
Comment (fyrchik) [reply README.md]: Fixed.

File: README.ru.md (hunk 2 above)
Comment (fyrchik) [reply README.ru.md:482]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**